### PR TITLE
Add SH19 dynamic confirmation text

### PR DESCRIPTION
**Pull request by HarryBosworth**
- Added section about SH19 same day is processed on the same day
- Removed corona delay warning when uploading SH19 using same day service.

Added a field to the confirmation page model indicating whether form type is sh19 same day form type.
Assumes SH19 same day service uses form type `SH19_SAMEDAY`

Resolves: [BI-8063](https://companieshouse.atlassian.net/browse/BI-8063)

### CODE REVIEW

**Review by YiannisPapastergiou — APPROVED** [review]

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -392,6 +392,7 @@ confirmation.next.fee.emailPending=We\u2019ll send you an email to confirm that
 confirmation.next.body1=An examiner will process the uploaded document to check if it\u2019s acceptable for filing. Documents are registered with the date they are uploaded to Companies House as long as they are acceptable.
 confirmation.next.fee.body1=An examiner will then process the uploaded document to check if it\u2019s acceptable for filing. Documents are registered with the date they are uploaded to Companies House as long as they are acceptable.
 confirmation.next.body2=If the document is acceptable for filing, we\u2019ll send a letter to tell you when we have started to review your case and explain what happens next. If the document is not acceptable, we\u2019ll email you to let you know what to do next.
+confirmation.sh19.same.day=You uploaded the SH19 using the same day service. This means the document is processed on the same working day if we receive it by 11am, and you’ve also uploaded the relevant resolution and solvency statement by this time. If you uploaded the SH19 and relevant documents after 11am, we will process them the next working day.
```
Comment (PhilipGlover) [inline src/main/resources/messages.properties:395]: apostrophe in you've is causing a display error
Screenshot 2021-07-01 at 13 38 47

**Review by PhilipGlover — CHANGES_REQUESTED** [review]
- conditions to display or not the appropriate text work, just the apostrophe to amend before approval
- sonar is happy, and test coverage is still 100%

File: src/main/resources/messages.properties (hunk 1 above)
Comment (PhilipGlover) [reply src/main/resources/messages.properties]: looking ok now

**Review by PhilipGlover — APPROVED** [review]
as per my earlier comment, run and displays as expected for SH19 stndard and Same day

**Pull request merged**